src/main.ts

```diff
@@ -1,11 +1,12 @@
 import { createApp } from 'vue'
 import App from './App.vue'
-import router from './libraries/router'
-import store from './libraries/store'
-const _ = require('lodash')
-import './libraries/axios'
 import i18n from './libraries/i18n'
+import store from './libraries/store'
+import router from './libraries/router'
+import _ from 'lodash';
 import langDetector from './libraries/i18n/langDetector'
-import './assets/css/style.css';
+import './libraries/axios'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './assets/css/style.css'
 
 createApp(App).use(i18n).use(store).use(router).use(_).use(langDetector).mount('#app')
```

Comment:
`.use(_)`
doesn't work properly, it's necessary to add the line:
`(window as any)._ = _`
It works for me